packages/constellation/jest.config.ts

```diff
@@ -128,13 +130,13 @@ export default {
   // runner: "jest-runner",
 
   // The paths to modules that run some code to configure or set up the testing environment before each test
-  // setupFiles: [],
+  setupFiles: ['./jest.setup.ts'],
 
   // A list of paths to modules that run some code to configure or set up the testing framework before each test
   setupFilesAfterEnv: [
     './src/setupTests.ts',
-    './mocks/localStorage.js',
-    './mocks/dispatchEvent.js',
+    // './mocks/localStorage.js',
```

Comment:
need these commented or can we just delete these two lines?

Reply:
Oh nice catch, will remove these.